### PR TITLE
Update UG Glossary and FAQ

### PR DESCRIPTION
**Pull request by shuyangk**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -98,6 +99,7 @@ Keep an eye out for them!
 --------------------------------------------------------------------------------------------------------------------
 
 ## Features
+[Back to Table of Contents](#table-of-contents)
```
Comment (leawang0) [inline docs/UserGuide.md]: I think it's better if we can put this at the end of every section. Actually I have added the link back to ToC in my other PR that is merged just now haha so I think we probably don't need this right now?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -143,6 +145,7 @@ add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​  [i/INSTRUMENT]…​  [g/GENRE]
 
 --------------------------------------------------------------------------------------------------------------------
 ### Features for managing musicians
+[Back to Table of Contents](#table-of-contents)
```
Comment (leawang0) [inline docs/UserGuide.md]: Same for this~

File: docs/UserGuide.md [hunk 3]
```diff
@@ -594,18 +600,60 @@ If you wish, please refer to the below section for a more succinct [command summ
...
-
-[to be added soon]
-
+[Back to Table of Contents](#table-of-contents)
+
+**Q**: How do I install Java 11?<br>
+**A**: Follow this [link](https://docs.oracle.com/en/java/javase/11/install/overview-jdk-installation.html#GUID-8677A77F-231A-40F7-98B9-1FD0B48C346A) for steps to download Java 11.
+
+**Q**: How do I transfer my data to another Computer?<br>
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous Addressbook home folder.
```
Comment (leawang0) [inline docs/UserGuide.md]: LGTM!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -594,18 +600,60 @@ If you wish, please refer to the below section for a more succinct [command summ
...
+| CLI       | Command Line Interface (CLI) is a text-based user interface that allows users to interact with a computer program by typing in commands through a keyboard. <br/> Since a mouse is not needed for CLI, it is often **a more efficient interaction method for fast typists.** |
+| GUI       | Graphical User Interface (GUI) is a user interface that allows users to interact with the application using graphical elements like text fields, buttons, and menus.                                                                                                         |
+| Parameter | Parameters are specific settings for you to customise your command. For example, the name and phone number of a musician are parameters.                                                                                                                                     |
+| Command   | A command is an instruction given to the application to perform a specific action. For example, `list` is a command that displays all musicians and bands.                                                                                                                   |
```
Comment (leawang0) [inline docs/UserGuide.md]: I think the glossary part also has merge conflict with my previous PR, but I think yours is better so feel free to replace mine!

**Review by leawang0 — CHANGES_REQUESTED** [review]

**Comment by tanyyyming** [comment]
All the checks fail because of the EOF newline not present in DG. Not a big issue, will fix on my side when I do the next PR.

**Pull request merged**